README.md

```diff
@@ -144,4 +146,42 @@ with f:
     f.search(doc_query, parameters={'filter': filter})
 ```
 
+### Limit returning results
+
+In some cases, you will want to limit the total number of retrieved results. `ElasticSearchIndexer` uses the `limit` argument 
+from the `match` function to set this limit. Note that when using `shards=N`, the `limit=K` is the number of retrieved results for **each shard** and total number of retrieved results is `N*K`. By default, `limits` is set to `20`. For more information about shards, please read [Jina Documentation](https://docs.jina.ai/fundamentals/flow/topology/#partition-data-by-using-shards)
+
+```python
+f =  Flow().add(
+    uses='jinahub://ElasticSearchIndexer',
+    uses_with={'match_args': {'limit': 10}})
+```
+
+### Configure the other search behaviors
+
+You can use `match_args` argument to pass arguments to the `match` function as below.
```

Comment:
```suggestion
You can use `match_args` argument to pass arguments to the `match` function as below.
```
let's mention `/search` endpoint as well so that people know we are referring to this endpoint

Reply:
it's added below in the  "Configure the Search Behaviors on-the-fly" section

Reply:
no I mean in this section you do not mention that the `/search` endpoint by this parameter.
From a user's perspective, he only needs to see the executor in https://hub.jina.ai/executor/2gyoad6q
so he doesn't care about how was this implemented (with match or so)